kedro-telemetry/kedro_telemetry/plugin.py

```diff
@@ -99,28 +115,90 @@ def before_command_run(
             )
 
 
-def _format_user_cli_data(
-    hashed_username: str, command_args: List[str], project_metadata: ProjectMetadata
+class KedroTelemetryHooks:  # pylint: disable=too-few-public-methods
```

Comment:
```suggestion
class KedroTelemetryProjectHooks:  # pylint: disable=too-few-public-methods
```

Minor: but this might make a clearer distinction between these hooks and the CLI hooks.